modules/generate-network-policies-ng20.adoc

```diff
@@ -8,14 +8,23 @@
+You can generate policies based on the cluster, namespaces, and deployments that you have selected in the network graph. Policies are generated for any deployments that are included in the current Network Graph scope. For example, the current scope could include the entire cluster, a cluster and namespaces, or individually selected deployments in the selected namespaces. You can also further reduce the scope by applying one of the filters from the *Filter deployments* field with any combination of the cluster, namespace, and deployment selections. For example, you could narrow the scope to deployments in a specific cluster and namespace that are affected by a specific CVE. Policies are generated from the traffic observed during the baseline discovery period.
 
 .Procedure
 . In the {product-title-short} portal, navigate to *Network Graph*.
 . Select a cluster, and then select one or more namespaces.
-. In the network graph header, select *Simulate network policy*. {product-title-short} generates policies for all of the deployments that exist in the selected cluster.
-. Optional: In the information panel that opens, select *Exclude ports & protocols* if you do not want ports and protocols to be scoped in {product-title-short} generated policies.
-. Select *Generate and simulate network policies*.
-The generated network policy configuration YAML file opens in the same panel, and the network graph shows the effects of the policies.
+. Optional: Select individual deployments to restrict the policy generated to just those deployments. You can also use the *Filter deployments* feature to further narrow the scope.
+. In the network graph header, select *Network policy generator*.
+. Optional: In the information panel that opens, select *Exclude ports & protocols* to remove the port/protocol restrictions when generating network policies from a baseline. For example, the `nginx3` deployment makes a port 80 connection to `nginx4`, and this is included as part of the baseline for nginx4. If policies are generated and this checkbox is not selected (the default behavior), the generated policy will restrict the allowed connections from `nginx3` to `nginx4` to only port 80. If policies are generated with this option selected, the generated policy will allow any port in the connection from `nginx3` to `nginx4`.
+. Click *Generate and simulate network policies*. {product-title-short} generates policies for the scope that you have chosen. This scope is displayed at the top of the *Generate network policies* panel. Clicking on the deployment information in the scope displays a list of the deployments that are included.
+. The generated network policy configuration YAML file opens in the same panel. You can copy the displayed YAML file to the clipboard or download it by clicking the download icon in the panel.
```

Comment:
I think there should be some sort of imperative here if this is going to be a step. I'd suggest choosing one of two options to modify this step:

1. Making this step a NOTE if this is just something that might be useful to know
2. Rewording it to something like this if it's an optional thing:
```suggestion
. Optional: Copy the generated network policy configuration YAML file to the clipboard or download it by clicking the download icon in the panel.
```